lib/order.rb

```diff
@@ -0,0 +1,45 @@
+class Order
+
+  DEFAULT_QUANTITY = 1
+
+  attr_reader :basket
+
+  def initialize
+    @basket = Hash.new
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)